cg/cli/deliver.py

```diff
@@ -17,46 +16,48 @@
 @click.pass_context
 def deliver(context):
     """Deliver stuff."""
-    pass
+    context.obj['db'] = Store(context.obj['database'])
+    context.obj['api'] = DeliverAPI(
```

Comment:
Can we name the key 'deliver_api'? It prevents me having to read code and then scroll up to see what API we're actually referring to.

Reply:
That change makes for too long lines, Isn't it quite safe to assume that an api in whatever.py is an whatever_api?

Reply:
The (additional) problem comes from that we initialize context.obj with the config (cg.yaml), which holds keys that might be overwritten if we are not careful. Even though this is a red herring in this topic it's got nothing to do with the 'api' key.

And yes, it is quite safe. My point was that for me as a person I see 'api' in all these files and I always have to know either the filename, look at the class signature, or look at the constructor to find out to what api we are referring to. Which I find unclear.

Reply:
fixed